app/views/recipe/index.html.erb

```diff
@@ -0,0 +1,31 @@
+    <%= stylesheet_link_tag "food", "data-turbo-track": "reload" %>
+        <%= stylesheet_link_tag "recipe", "data-turbo-track": "reload" %>
+<h1>My Recipes</h1>
```

Comment:
[REQUIRED] Kindly remove the h1 tag as it is not in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_index.png?token=GHSAT0AAAAAACHZNTDRFNRO7J7NLQCBR5TIZJJ3H4A).